src/common/constants.js

```diff
@@ -54,11 +54,21 @@ const dischargeTypes = [
 
 const suffixes = ['Jr.', 'Sr.', 'II', 'III', 'IV'];
 
-const genders = [
+const birth_sexes = [
```

Comment:
I am worried updating this variable is going to cause issues with other forms since they all pretty much use this constant. We can probably make this backwards compatible by leaving this `gender` and updating to `birthSex` on frontend and we can still pass as `birth_sex` to BE.